### PR TITLE
Corrects the C02 to CO2 for the CO2 knife

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Dicarbon is in fact not Carbon Dioxide

# Explain why it's good for the game

it fixes a typo

# Changelog

:cl:
spellcheck: Corrects the C02 Knife to CO2
/:cl:

### CODE REVIEW

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
`CO2` -> `co2`

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -367,9 +367,9 @@ Defined in conflicts.dm of the #defines folder.
 		return
 	..()
 
-/obj/item/c02_cartridge //where tf else am I gonna put this?
-	name = "C02 cartridge"
-	desc = "A cartridge of compressed C02 for the M8 cartridge bayonet. Do not consume or puncture."
+/obj/item/co2_cartridge //where tf else am I gonna put this?
+	name = "CO2 cartridge"
```
Comment (Zonespace27) [inline code/modules/projectiles/gun_attachables.dm:371]: improper

File: code/modules/client/preferences_gear.dm [hunk 2]
```diff
@@ -396,7 +396,7 @@ var/global/list/gear_datums = list()
 
 /datum/gear/m8_cartridge_bayonet
 	display_name = "M8 Cartridge Bayonet"
-	path = /obj/item/storage/box/c02_knife
+	path = /obj/item/storage/box/CO2_knife
```
Comment (Zonespace27) [inline code/modules/client/preferences_gear.dm:399]: lower case

File: code/game/objects/items/storage/misc.dm [hunk 3]
```diff
@@ -124,17 +124,17 @@
...
 	foldable = TRUE
 	storage_slots = 3
 	w_class = SIZE_SMALL
 	max_w_class = SIZE_SMALL
 
-/obj/item/storage/box/c02_knife/fill_preset_inventory()
-	new /obj/item/attachable/bayonet/c02(src)
-	new /obj/item/c02_cartridge(src)
-	new /obj/item/c02_cartridge(src)
+/obj/item/storage/box/CO2_knife/fill_preset_inventory()
```
Comment (Zonespace27) [inline code/game/objects/items/storage/misc.dm:137]: lower case

File: code/datums/supply_packs/black_market.dm [hunk 4]
```diff
@@ -965,7 +965,7 @@ This is where the RO can reclaim their lost honor and purchase the M44 custom, t
 
 /datum/supply_packs/contraband/deep_storage/cartridge_bayonet
 	name = "M8 Cartridge Bayonet Kit"
-	contains = list(/obj/item/storage/box/c02_knife)
+	contains = list(/obj/item/storage/box/CO2_knife)
```
Comment (Zonespace27) [inline code/datums/supply_packs/black_market.dm:968]: lower case

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by blackcrystall** [comment]
Why do you think that incorrect name of knife (C02)???

**Pull request merged**